subprojects/gdk_rust/gdk_electrum/src/store.rs

```diff
@@ -87,19 +84,19 @@ pub struct RawStore {
     /// wallet settings
     settings: Option<Settings>,
 
-    /// transaction memos
-    memos: HashMap<bitcoin::Txid, String>,
+    /// account names
+    account_names: HashMap<u32, String>,
+
+    /// transaction memos (account_num -> txid -> memo)
+    memos: HashMap<u32, HashMap<bitcoin::Txid, String>>,
```

Comment:
As discussed separately, for consistency with green-cpp, memos should be per tx,
without discriminating per subaccount. We also need a test for a memo associated 
to a tx for 2 subaccounts in the same wallet, for testing regression in this context.

Reply:
Addressed by https://github.com/Blockstream/gdk/pull/93/commits/0ff9b55bb3522bcb7b5dbc517c6962f478854766.